File: apps/problems/two-sum/boilerplate-full/function.java

      import java.io.*;
      import java.util.*;
  
      //##USER_CODE_HERE##
  
      public class Main {
        
      public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int num1 = Integer.parseInt(br.readLine().trim());
int num2 = Integer.parseInt(br.readLine().trim());
        Solution obj = new Solution();
        int result = obj.sum(num1, num2);
        System.out.println(result);
        br.close();
      }
      }
    

File: apps/problems/two-sum/boilerplate-full/function.cpp
#include <iostream>
#include <fstream>
#include <vector>
#include <string>
#include <sstream>
#include <climits>

//##USER_CODE_HERE##


int main() {
    int num1;
std::cin >> num1;
int num2;
std::cin >> num2;
    Solution obj;
    int result = obj.sum(num1, num2);
    std::cout << result << std::endl;
    return 0;
}
